### DISCUSSION TITLE
Cambio en unidades de conservación para identificar documentos físicos y digitales desde el registro de archivos

### DISCUSSION THREAD

**Discussion by NestorFGG** [SINCOSOFT-SAS/SGD#39] [General]
Esta funcionalidad permitirá independizar los tipo de almacenamiento de archivo definidos en las TRD, para trasladarse a unidades de conservación y con ellos establecer condiciones de expedientes híbridos de archivo.

**Comment by aleja550** [comment]
Nunca se realizo ni tampoco está en planeación del producto
